### PR TITLE
WIP: Updates to Vivian's Bio

### PR DESCRIPTION
**Pull request by vivianlin2000**

### CODE REVIEW

**Comment by vivianlin2000** [comment]
@sbillinge, this is ready for review and merge

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1 @@
-.idea
-_build
+.DS_Store
```
Comment (sbillinge) [inline .gitignore:1]: this needs a carriage return (put CR at end of files)

Also, this .gitignore shouldn't be in the db directory.  I think you will find one already in the top directory that this entry should be added to.

File: db/people.yml [hunk 2]
```diff
@@ -3106,10 +3106,19 @@ vgharakhanyan:
   position: Graduate Student
 vlin:
   active: true
-  aka: []
+  aka: Viv
```
Comment (sbillinge) [inline db/people.yml:3109]: these aka's are used for linking you to your publications etc., so if there is a paper that is published where you appear as "Lin, V." we wouldn't be able to correctly dereference it to your profile here where you appear as either vlin or "Vivian Lin".  Just leave this as an empty list until this becomes an issue! ;)

File: db/people.yml [hunk 3]
```diff
@@ -3106,10 +3106,19 @@ vgharakhanyan:
...
   avatar: https://github.com/vivianlin2000.png
-  bio: bio
-  education: []
+  bio: Vivian is an undergraduate researcher at Columbia University. She was bornand raised in
```
Comment (sbillinge) [inline db/people.yml:3111]: space in bornand

File: .gitignore (hunk 1 above)
Comment (vivianlin2000) [reply .gitignore:1]: so should I delete this .gitignore file? I'm a little confused as how to get the pre-existing .gitignore file and add to that.

File: db/people.yml (hunk 2 above)
Comment (vivianlin2000) [reply db/people.yml]: fixed!

**Comment by sbillinge** [comment]
Great job @vivianlin2000 , great sleuthing!!!!!!

**Pull request merged**